src/popup/utils/utils.ts

```diff
@@ -503,15 +506,14 @@ export function truncateString(text: string, maxLength: number) {
 }
 
 export function amountRounded(rawAmount: number | BigNumberPublic): string {
-  let amount = rawAmount;
-  if (typeof rawAmount !== 'object') {
-    amount = new BigNumber(rawAmount);
-  }
+  const amount: BigNumberPublic = (typeof rawAmount === 'object')
+    ? rawAmount
+    : new BigNumber(rawAmount);
 
-  if (amount < 0.01 && amount.toString().length < 9 + 2) {
+  if (amount.isLessThan(0.01) && amount.toString().length < 9 + 2) {
     return amount.toFixed();
   }
-  return amount.toFixed((amount < 0.01) ? 9 : 2);
+  return amount.toFixed((amount.isLessThan(0.01)) ? 9 : 2);
```

Comment:
Are the nines related to one another? If so maybe they could be put in a variable

Reply:
OK, I introduced custom constants for this.